bin/aurbuild

```diff
@@ -133,6 +133,10 @@ if [[ $build == build_chroot ]]; then
     sudo cp -v "$tmp"/pacman.conf "$CHROOT"/root/etc/pacman.conf
 fi
 
+# Canonicalise paths
```

Comment:
I'd summarize the commit here, something like "Avoid issues with relative paths", as everybody knows what realpath does, but perhaps not why you've used it here.